solara/validate_hooks.py

```diff
@@ -168,8 +168,8 @@ def error_on_invalid_scope(self, use_node: ast.Call, use_node_id: str):
         scope_line = self.outer_scope.lineno + self.line_offset
         raise HookValidationError(
             cause,
-            f"{self.get_function_name()}: `{use_node_id}` found on line" f" {offset_use} within a {cause.value} created on line {scope_line}",
+            f"{self.get_source_context(offset_use)}: `{use_node_id}` found on line within a {cause.value} created on line {scope_line}",
```

Comment:
You can probably remove "found on line" from this statement if we include line number in the link

Reply:
Good catch, done!